python/sglang/srt/layers/attention/flashinfer_backend.py

```diff
@@ -339,9 +340,10 @@ def init_forward_metadata_replay_cuda_graph(
         req_pool_indices: torch.Tensor,
         seq_lens: torch.Tensor,
         seq_lens_sum: int,
-        encoder_lens: Optional[torch.Tensor],
         forward_mode: ForwardMode,
+        encoder_lens: Optional[torch.Tensor],
         spec_info: Optional[SpecInfo],
+        **kwargs,
```

Comment:
Do not use `kwargs`. It makes the code more unreadable because we do not know what exact arguments are.
Is it possible to specify it more clearly?